model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -55,7 +67,7 @@ class OrtConfigurationTest : WordSpec({
 
                         backend.localFileStorage.let { localFileStorage ->
                             localFileStorage shouldNotBe null
-                            localFileStorage!!.directory shouldBe File("~/.ort/scanner")
+                            localFileStorage!!.directory shouldBe File("~/.ort/scanner/results")
```

Comment:
Shouldn't this be part of https://github.com/heremaps/oss-review-toolkit/pull/1899?

Reply:
Added to #1899, I will rebase this PR later.